fitbenchmarking/fitbenchmarking
# Setting Jacobian in Mantid controller

#### Description of Work

Fixes #485 

For none Mantid data the Mantid controller will now use the user defined Jacobian from the options.

#### Testing Instructions

1. Check the `chi_sq` values are consistent between this branch and master
2. Check that the order of the two loops in `functionDeriv1D` does not matter with respect to time and `chi_sq`


Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
